Clinical-Genomics/cgweb
# handle external like mip

This fix makes OP handle external samples like mip samples

How to test:
install on stage
create external OP order

Expected result:
panels visible for families

- [x] code reviewed by AJ 
- [x] test @patrikgrenfeldt 
- [x] deploy ready PG

this is a fix-patch
